js/views/components/PageControls.js

```diff
@@ -34,7 +34,9 @@ export default class extends baseVw {
   }
 
   render() {
-    loadTemplate('components/pageControls.html', (t) => {
+    const path = this.getState().type === 'search' ?
+      'components/pageControlsSearch.html' : 'components/pageControls.html';
+    loadTemplate(path, (t) => {
```

Comment:
Can we actually make a seperate view for the other page controls? I'm not sure why we'd want them to share one view.

Also, I'm not sure if `pageControlsSearch` is the best name for the second component since we may want to use it in other areas than search. Unless by that name, you mean it's "search style" page controls. I guess I could buy that if that's your intent.

Reply:
I couldn't think of a good abstract name for it. Maybe pageControlsTextStyle?

Re: view, the view functionality is literally identical other than the template, having 2 seems like unnecessary duplication to me.